### PR TITLE
Fix at least one issue with overflow on rotation

### PR DESCRIPTION
**Pull request by cnlohr**

### CODE REVIEW

File: main/modes/mode_flight.c [hunk 1]
```diff
@@ -1223,9 +1223,9 @@ static void flightGameUpdate( flight_t * tflight )
         tflight->hpr[0] += tflight->pitchmoment;
         tflight->hpr[1] += tflight->yawmoment;
         
-        if( tflight->hpr[0] > 3960 ) tflight->hpr[0] -= 3960;
+        if( tflight->hpr[0] >= 3960 ) tflight->hpr[0] -= 3960;
```
Comment (AEFeinstein) [inline main/modes/mode_flight.c:1226]: This makes a difference? `sin1024[]` is accessed through `getSin1024()`, which should mod the value down to the proper range anyway.
```C
int16_t getSin1024(int16_t degree)
{
    degree = ( (degree % 360) + 360 ) % 360;
    return sin1024[degree];
}
```

Comment (AEFeinstein) [inline main/modes/mode_flight.c:1226]: This makes a difference? `sin1024[]` is accessed through `getSin1024()`, which should mod the value down to the proper range anyway.
```C
int16_t getSin1024(int16_t degree)
{
    degree = ( (degree % 360) + 360 ) % 360;
    return sin1024[degree];
}
```

Comment (AEFeinstein) [reply main/modes/mode_flight.c:1226]: Ah, I guess the values are used here too, and not properly bounded
```C
tdRotateEA( ProjectionMatrix, tflight->hpr[1]/11, tflight->hpr[0]/11, 0 );
```

**Review by AEFeinstein — APPROVED** [review]

**Comment by AEFeinstein** [comment]
@cnlohr if you're satisfied, delete the branch and close the ticket, otherwise I'll assume you're still tinkering.

**Comment by cnlohr** [comment]
branch deleted.

**Pull request merged**